ansible/roles/dashd/templates/dash.conf.j2

```diff
@@ -44,6 +44,7 @@ rpcallowip=0.0.0.0/0
 rpcallowip={{ dashd_private_cidr }}
 rpcallowip=127.0.0.0
 rpcallowip=172.0.0.0/8
```

Comment:
Here should be 172.16.0.0/12 as rest of network is routable to real internet

Reply:
Sometimes docker interface is `172.17...` or even different, no?

Reply:
Since this runs within Docker I'd recommend setting this (only) to `0.0.0.0/0` and letting the host handle the infra. Networking within Docker doesn't mean that this is available outside the Docker network, only to all hosts in the network this container is running.

Reply:
It runs using host network https://github.com/dashpay/dash-network-deploy/blob/48c535806218310a4a3bb12f3b538175401ae9b2/ansible/roles/dashd/tasks/main.yml#L52